storybook/storybook-react/src/PageGrid/PageGrid.stories.tsx

```diff
@@ -3,103 +3,95 @@
-      layout: 'centered',
-    },
-  },
 }
 
 export const ThreeColumns: Story = {
+  ...StoryTemplate,
   args: {
     children: [
-      <GridCell key={1} gridColumns={4}>
-        <div className="amsterdam-docs-pink-box">
-          <Paragraph>Blok 1: kolom 1 t/m 4</Paragraph>
-        </div>
-      </GridCell>,
-      <GridCell key={2} gridColumns={4}>
-        <div className="amsterdam-docs-pink-box">
-          <Paragraph>Blok 2: kolom 5 t/m 8</Paragraph>
-        </div>
-      </GridCell>,
-      <GridCell key={3} gridColumns={4}>
-        <div className="amsterdam-docs-pink-box">
-          <Paragraph>Blok 3: kolom 9 t/m 12</Paragraph>
-        </div>
-      </GridCell>,
-    ],
-  },
-}
-
-export const ThreeImages: Story = {
-  args: {
-    children: [
-      <GridCell key={1} gridColumns={4}>
+      <GridCell key={1} span={4}>
         <figure className="amsterdam-docs-figure">
-          <img src="https://picsum.photos/800/450?random=1" />
+          <img alt="Voorbeeld van een afbeelding" src="https://picsum.photos/1024/576?random=1" />
         </figure>
       </GridCell>,
-      <GridCell key={2} gridColumns={4}>
+      <GridCell key={2} span={4}>
         <figure className="amsterdam-docs-figure">
-          <img src="https://picsum.photos/800/450?random=2" />
+          <img alt="Voorbeeld van een afbeelding" src="https://picsum.photos/1024/576?random=2" />
         </figure>
       </GridCell>,
-      <GridCell key={3} gridColumns={4}>
+      <GridCell key={3} span={4}>
         <figure className="amsterdam-docs-figure">
-          <img src="https://picsum.photos/800/450?random=3" />
+          <img alt="Voorbeeld van een afbeelding" src="https://picsum.photos/1024/576?random=3" />
         </figure>
       </GridCell>,
     ],
+    title: 'Drie afbeeldingen',
   },
 }
 
-export const MiddleBlock: Story = {
+export const FullWidth: Story = {
+  ...StoryTemplate,
   args: {
     children: (
-      <GridCell gridColumns={{ start: 2, span: 10 }}>
+      <GridCell fullWidth>
         <div className="amsterdam-docs-pink-box">
-          <Paragraph>Blok: kolom 2 t/m 11</Paragraph>
+          <Paragraph>Full width</Paragraph>
         </div>
       </GridCell>
     ),
   },
 }
 
-export const Paragraphs: Story = {
+export const MiddleBlock: Story = {
+  ...StoryTemplate,
   args: {
-    children: [
-      <Paragraph key={1} gridColumns={{ start: 1, span: 3 }}>
-        A paragraph, directly on the grid, with a width of 3 columns.
-      </Paragraph>,
-      <Paragraph key={2} gridColumns={{ start: 4, span: 9 }}>
-        A paragraph, directly on the grid, with a width of 9 columns. Lorem ipsum dolor sit amet, consectetur
-        adipisicing elit. Eligendi id quibusdam soluta. Architecto aspernatur at debitis, deserunt ea error ex impedit
-        magnam nihil odio perferendis placeat porro quidem repellat temporibus ut voluptate.
-      </Paragraph>,
-    ],
+    children: (
+      <GridCell start={[1, 2, 2]} span={[4, 6, 10]}>
```

Comment:
I don't think this api is very intuitive. What about `start={{ small: 1, medium: 2, large: 2 }}` etc.?